### DISCUSSION TITLE
How to compile the teleport community version？

### DISCUSSION THREAD

**Discussion by RyanCode** [gravitational/teleport#12628] [Q&A] [answered]
>Trying out compile Teleport locally? always need a license to start it.How to build the community version locally？
```shell
sh-3.2$ sudo teleport configure -o file --cluster-name=tele.example.com --public-addr=tele.example.com:443 --cert-file=/var/lib/teleport/fullchain.pem --key-file=/var/lib/teleport/privkey.pem
Password:
Wrote config to file "/etc/teleport.yaml". Add your license file to /var/lib/teleport/license.pem and start the server. Happy Teleporting!
sh-3.2$

**Comment by webvictim** [comment]
What does `teleport version` say?

You can download the open source binaries from https://goteleport.com/download.

The Enterprise binaries are served from a different URL.

**Reply by RyanCode** [reply]
Thanks for the answer, teleport version
```shell
Teleport v8.0.0 git:v8.0.0-0-g6720c97e9 go1.18.2
```
I noticed that this link does not have the **source code** for the Community Edition.but [Install from Source](https://goteleport.com/installing),and I checkout to stable version of v8.0.0,Unfortunately, I still need to import licenses after the build is complete

**Comment by webvictim** [comment, marked as answer]
To actually answer your question:

```console
git clone https://github.com/gravitational/teleport
cd teleport 
git checkout v9.2.3 # replace with tag for the Teleport version you want to compile
make release
```

You'll need Go 1.17.10 installed and probably a few other dependencies.

You can also build in a Docker container (which will handle all dependencies etc for you) using `make -C build.assets release` instead.

**Reply by RyanCode** [reply]
Thanks a lot！
